sdk/monitor/azure-monitor-query/src/main/java/com/azure/monitor/query/package-info.java

```diff
@@ -2,6 +2,121 @@
+ *         .buildAsyncClient&#40;&#41;;
+ * </pre>
+ * <!-- end com.azure.monitor.query.LogsQueryAsyncClient.instantiation -->
+ *
+ * <p>The following code sample demonstrates the creation of a {@link com.azure.monitor.query.MetricsQueryAsyncClient}
+ * using the {@link com.azure.monitor.query.MetricsQueryClientBuilder}.</p>
+ *
+ * <!-- src_embed com.azure.monitor.query.MetricsQueryAsyncClient.instantiation -->
+ * <pre>
+ * MetricsQueryAsyncClient metricsQueryAsyncClient = new MetricsQueryClientBuilder&#40;&#41;
+ *         .credential&#40;tokenCredential&#41;
+ *         .buildAsyncClient&#40;&#41;;
+ * </pre>
+ * <!-- end com.azure.monitor.query.MetricsQueryAsyncClient.instantiation -->
+ *
+ * <p><strong>Sample: Construct Synchronous Clients</strong></p>
+ *
+ * <p>The following code sample demonstrates the creation of a {@link com.azure.monitor.query.LogsQueryClient}
+ * using the {@link com.azure.monitor.query.LogsQueryClientBuilder}.</p>
+ *
+ * <!-- src_embed com.azure.monitor.query.LogsQueryClient.instantiation -->
+ * <pre>
+ * LogsQueryClient logsQueryClient = new LogsQueryClientBuilder&#40;&#41;
+ *         .credential&#40;tokenCredential&#41;
+ *         .buildClient&#40;&#41;;
+ * </pre>
+ * <!-- end com.azure.monitor.query.LogsQueryClient.instantiation -->
+ *
+ * <p>The following code sample demonstrates the creation of a {@link com.azure.monitor.query.MetricsQueryClient}
+ * using the {@link com.azure.monitor.query.MetricsQueryClientBuilder}.</p>
+ *
+ * <!-- src_embed com.azure.monitor.query.MetricsQueryClient.instantiation -->
+ * <pre>
+ * MetricsQueryClient metricsQueryClient = new MetricsQueryClientBuilder&#40;&#41;
+ *         .credential&#40;tokenCredential&#41;
+ *         .buildClient&#40;&#41;;
+ * </pre>
+ * <!-- end com.azure.monitor.query.MetricsQueryClient.instantiation -->
+ *
+ * <br/>
+ *
+ * <hr/>
+ *
+ * <h2>Query Workspace</h2>
+ *
+ * <p>The {@link com.azure.monitor.query.LogsQueryClient#queryWorkspace(java.lang.String, java.lang.String, com.azure.monitor.query.models.QueryTimeInterval) Query Workspace API} method can be used to
+ * query logs from a given workspace.</p>
+ *
+ * <p>The sample below shows how to query logs from the last 24 hours</p>
+ *
+ * <!-- src_embed com.azure.monitor.query.LogsQueryClient.query#String-String-QueryTimeInterval -->
+ * <pre>
+ * LogsQueryResult queryResult = logsQueryClient.queryWorkspace&#40;&quot;&#123;workspace-id&#125;&quot;, &quot;&#123;kusto-query&#125;&quot;,
+ *         QueryTimeInterval.LAST_DAY&#41;;
+ * for &#40;LogsTableRow row : queryResult.getTable&#40;&#41;.getRows&#40;&#41;&#41; &#123;
+ *     System.out.println&#40;row.getRow&#40;&#41;
+ *             .stream&#40;&#41;
+ *             .map&#40;LogsTableCell::getValueAsString&#41;
+ *             .collect&#40;Collectors.joining&#40;&quot;,&quot;&#41;&#41;&#41;;
+ * &#125;
+ * </pre>
+ * <!-- end com.azure.monitor.query.LogsQueryClient.query#String-String-QueryTimeInterval -->
```

Comment:
We should have a sample for querying metrics too if we are going to add a sample for querying logs here.